R/variance_thresholding.R

```diff
@@ -22,15 +22,14 @@ variance_thresholding <- function(data, threshold = 0) {
   }
 
   selected_feature_indexes <- c()
-  i <- 1
-  for (name in names(data)) {
+  for (i in 1:length(names(data))) {
+    # Get variance of a column
     var_i <- stats::var(data[,i])
 
     if (var_i > threshold) {
+      # Add the column to the selected pile if variance's above the threshold
```

Comment:
> ... if variance's above ...

Does this mean "if variance is above"? If so, can you change it to avoid confusion?

Reply:
Ok, I see the fix. Thanks.